content/docs/guides/webassembly/resources.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Additional Resources"
+weight: 4
+description: |
+  Additional Resources for using TinyGo with WASM/WASI
+---
+
+Here are a few resources for helping learn more about using TinyGo with WebAssembly.
+
+**Wasm By Example**
+https://wasmbyexample.dev/
+
+**Are We Wasm Yet ? - Part 1**
+https://elewis.dev/are-we-wasm-yet-part-1
+
+**Are We Wasm Yet ? - Part 2**
+https://elewis.dev/are-we-wasm-yet-part-2
+
+**Shrink Your TinyGo WebAssembly Modules by 60%**
+https://www.fermyon.com/blog/optimizing-tinygo-wasm
+
+**Writing a WebAssembly Service in TinyGo for Wagi and Spin**
+https://www.fermyon.com/blog/tinygo-webassembly-favicon-server
+
+**wazero - TinyGo**
+https://wazero.io/languages/tinygo/
+
+**WasmEdge Runtime - Go**
+https://wasmedge.org/book/en/write_wasm/go.html
+
+**An essay on the bi-directional exchange of strings between the Wasm module (with TinyGo) and Node.js (with WASI support)**
+https://www.wasm.builders/k33g_org/an-essay-on-the-bi-directional-exchange-of-strings-between-the-wasm-module-with-tinygo-and-nodejs-with-wasi-support-3i9h
```

Comment:
This post contains some incorrect examples. For example:

```go
//export hello 
func hello() *byte {
  return &(([]byte)("hello world")[0]) 
}
```

This code provides no way to prevent the returned object from being garbage collected. It should use `malloc` or at least store the value in a global.

Reply:
@deadprogram it's an old blog post; this one is perhaps better https://k33g.hashnode.dev/wasi-communication-between-nodejs-and-wasm-modules-with-the-wasm-buffer-memory#heading-calling-a-wasm-function-from-nodejs-with-strings

I am writing several series:
- https://k33g.hashnode.dev/series/wasi-nodejs
- https://k33g.hashnode.dev/series/wazero-first-steps

Reply:
Changed out for those new series which are :fire: great work @k33g thank you!!!